sds/src/proximity_keywords.rs

```diff
@@ -32,29 +32,37 @@ const EXCLUDED_KEYWORDS_REMOVED_CHARS: &[char] = &['-', '_'];
 /// Characters that are considered to be links between keyword words.
 /// Example: '-' in "aws-access" is considered to be a link character.
 /// Example: '.' in "my.path" is considered to be a link character.
-const MULTI_WORD_KEYWORDS_LINK_CHARS: &[char] = &['-', '_', '.', ' ', '/'];
+pub const MULTI_WORD_KEYWORDS_LINK_CHARS: &[char] = &['-', '_', '.', ' ', '/'];
 
-const UNIFIED_LINK_CHAR: char = '.';
+pub const UNIFIED_LINK_CHAR: char = '.';
 
 struct ProximityKeywordsRegex<const EXCLUDED_CHARS: bool> {
     content_regex: meta::Regex,
-    #[allow(dead_code)]
     path_regex: meta::Regex,
 }
 
 impl CompiledProximityKeywords {
-    pub fn is_false_positive_match(&self, content: &str, match_start: usize) -> bool {
+    pub fn is_false_positive_match(&self, content: &str, path: &str, match_start: usize) -> bool {
         match (
             &self.included_keywords_pattern,
             &self.excluded_keywords_pattern,
         ) {
             (Some(included_keywords), _) => {
-                let is_false_positive = !contains_keyword_match(
+                let is_valid_from_path = contains_keyword_in_path(path, included_keywords);
+
+                if is_valid_from_path {
+                    return false;
+                }
+
+                let is_valid_from_content = contains_keyword_match(
                     content,
                     match_start,
                     self.look_ahead_character_count,
                     included_keywords,
                 );
+
+                let is_false_positive = !(is_valid_from_path || is_valid_from_content);
```

Comment:
Here `is_valid_from_path` will be alway false, it can probably be simplified.

Reply:
[Remove useless variable](https://github.com/DataDog/dd-sensitive-data-scanner/pull/53/commits/59b7c183a9654134a6cd009f088310c5671745fb)